pkg/types/types.go

```diff
@@ -3,11 +3,17 @@ package types
 import dio "github.com/aquasecurity/go-dep-parser/pkg/io"
 
 type Library struct {
-	ID       string `json:",omitempty"`
-	Name     string
-	Version  string
-	Indirect bool   `json:",omitempty"`
-	License  string `json:",omitempty"`
+	ID                 string `json:",omitempty"`
+	Name               string
+	Version            string
+	Indirect           bool          `json:",omitempty"`
+	License            string        `json:",omitempty"`
+	ExternalReferences []ExternalRef `json:",omitempty"`
+}
+
+type ExternalRef struct {
+	Type RefType
+	Url  string
```

Comment:
https://github.com/golang/go/wiki/CodeReviewComments#initialisms

```suggestion
	URL  string
```